extensions/ql-vscode/src/view/common/Telemetry.ts

```diff
@@ -0,0 +1,56 @@
+import * as React from "react";
```

Comment:
Can we rename this file to `telemetry.ts`? The only files which are upper-cased contain React components, so for utility functions or hooks I'd expect to see a lower-case filename.

Usually, I'd also prefer the file to be named the same as the hook (so `useStateWithTelemetry.ts`), but since this also contains another function I don't think that makes sense in this case.